src/main/features/ccj/form/models/paidAmount.ts

```diff
@@ -17,20 +19,25 @@ export class PaidAmount implements Serializable <PaidAmount> {
   @ValidateIf(o => o.option === PaidAmountOption.YES)
   @IsDefined({ message: ValidationErrors.AMOUNT_REQUIRED })
   @IsPositive({ message: ValidationErrors.AMOUNT_NOT_VALID })
+  @IsLessThan('claimedAmount', { message: ValidationErrors.PAID_AMOUNT_GREATER_THAN_TOTAL_AMOUNT })
```

Comment:
This validator is gone on master, but you can probably restore it.
It was only used by repayment plan which is more complex so it has been changed to work with it.
I think if you just bring `IsLessThan` back it would be fine